js/foundation.tabs.js

```diff
@@ -69,8 +69,12 @@ class Tabs {
         'aria-labelledby': linkId
       });
 
-      if(isActive && _this.options.autoFocus){
-        $link.focus();
+      if(isActive && _this.options.autoFocus){  
+        $(window).load(function() {         
+          $('html, body').animate({ scrollTop: $elem.offset().top }, _this.options.deepLinkSmudgeDelay, () => {
+            $link.focus();
```

Comment:
If the focused element is under the fold, the browser will natively scroll down to reach that element, so if the active `tab` is under the fold, then the scrolling event will be fired twice, once as a native behavior and the second is ours. So I thought that it would be good to fire the `focus` event after the browser is already there, that will prevent the browser to do anymore scrolling.
To test this scenario:
- Create an active tab under the fold.
- Open the page.
- Refresh the page.